fission-cli/library/Fission/CLI/IPFS/Executable.hs

```diff
@@ -71,7 +71,7 @@ place' host = do
   IPFS.BinPath ipfsPath <- Path.globalIPFSBin
 
   -- Network
-  ipfsBin <- ensureM . unpack =<< download (IPFS.Version 0 9 0) host
+  ipfsBin <- ensureM . unpack =<< download (IPFS.Version 0 12 2) host
```

Comment:
Outside of the scope here (I think), but testing this I realized - we don't automatically update the ipfs binary for existing installs (you can force it by running `fission setup`) but with a nice new feature like this, might be something we consider in the future?

Reply:
@walkah could you write up an issue if one doesn't already exist? 🙏

Reply:
Will do!